website/docusaurus.config.js

```diff
@@ -64,13 +64,13 @@ var siteSettings = {
     announcementBar: {
       id: "live_qa",
       content:
-        "Register now for Coalesce 2022. The Analytics Engineering Conference!",
+        "Got opinions on analytics engineering? Tell us in the 5-minute dbt Community Survey.",
```

Comment:
change to sentence casing
```suggestion
        "Got opinions on analytics engineering? Tell us in the 5-minute dbt Community survey.",
```